docs/source/advanced_features/macro_system.rst

```diff
@@ -191,7 +191,7 @@ We can now observe the process of macro expansion in action:
 
 .. code:: python
 
-    from graphql_compiler.macros import perform_macro_expansion
+    from graphql_compiler.macros import get_schema_with_macros, perform_macro_expansion
```

Comment:
The changes to this file look great but they seem completely unrelated to this PR.

Reply:
Will remove.